### PR TITLE
lab 4-5

### PR DESCRIPTION
**Pull request by Osipchik**
[deploy link](https://spa-app-fa4b0.firebaseapp.com/)

I had some problems with deploying the application, so I created a [second repository](https://github.com/Tim-Osipchik/spa) with this project, everything worked out there.

Tthe code in both repositories is the same

The requirement to delete playlists has been fulfilled. I added the ability to delete a user's playlist, but not an artist's album.
Also, I added the ability to change the artist header image and background colors of playlist and artist page. 
Artist can't create an album until he updated info about himself

### CODE REVIEW

**Review by gmltA — CHANGES_REQUESTED** [review]
Can you publish your changes so I can see not the whole code, but the difference somehow?

**Comment by gmltA** [comment]
Also the music is not playing for me...

**Comment by Osipchik** [comment]
I've added the music player and removed the folder with styles they are the same as in the previous pr

File: public/utils/componentsHelper.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    const currentDate = new Date()
+    const currentHours = currentDate.getHours()
+    
+    if (currentHours < 12) {
+        return 'Доброе утро'
+    } else if (currentHours < 17) {
+        return 'Добрый день'
+    } else {
+        return 'Добрый вечер'
+    }
```
Comment (gmltA) [inline public/utils/componentsHelper.js:13]: 👍

**Review by gmltA — APPROVED** [review]
Looks solid.

That's an overall <kbd>9</kbd>.

Good luck!

**Pull request closed without merging**